multishard_mutation_query.cc

```diff
@@ -1001,13 +1001,13 @@ future<std::tuple<foreign_ptr<lw_shared_ptr<reconcilable_result>>, cache_tempera
 
 future<std::tuple<foreign_ptr<lw_shared_ptr<query::result>>, cache_temperature>> query_data_on_all_shards(
         distributed<replica::database>& db,
-        schema_ptr table_schema,
+        schema_ptr query_schema,
         const query::read_command& cmd,
         const dht::partition_range_vector& ranges,
         query::result_options opts,
         tracing::trace_state_ptr trace_state,
         db::timeout_clock::time_point timeout) {
-    schema_ptr query_schema = cmd.slice.is_reversed() ? table_schema->make_reversed() : table_schema;
+    schema_ptr table_schema = cmd.slice.is_reversed() ? query_schema->make_reversed() : query_schema;
```

Comment:
@denesb the notion of a query schema also applies to mutation fragment selection. It has the same keys as the base table but the columns of the mutation_fragment transformation. I don't think we should do anything now but keep it in mind if it helps one day.

Reply:
Same here (consider using `get_reversed()` instead of `make_reversed()`)